dashboard/app/controllers/pd/professional_learning_controller.rb

```diff
@@ -80,6 +80,15 @@ def csaif
     end
   end
 
+  # GET professional-learning/regional-partner/playbook
+  def rp_playbook
+    if current_user.permission?(UserPermission::PROGRAM_MANAGER) || current_user.permission?(UserPermission::WORKSHOP_ADMIN)
+      render 'pd/professional_learning/regional_partner/regional_partner_playbook'
+    else
+      render 'pd/professional_learning/regional_partner/not_permitted_to_view'
```

Comment:
related to @drizco's question, is it possible to return a 403 here and render this view? 🤔

Reply:
a very quick google suggests that maybe something like `render 'pd/professional_learning/regional_partner/not_permitted_to_view', :status => :forbidden`

This would (I think) prevent Google from indexing this page as they would view it as requiring sign in.

Reply:
Good call! Will do, and I'll update the other similar renders too

Reply:
![403](https://github.com/user-attachments/assets/2532e73f-4388-417d-a447-21df803ffeae)